pkgs/build-support/php/pkgs/composer-phar.nix

```
@@ -46,13 +50,18 @@ stdenvNoCC.mkDerivation (finalAttrs: {
     runHook postInstall
   '';
 
+  postInstall = ''
```

```suggestion
  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
```

This will make cross-compilation not break.